.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "rust-agent/grpc-rs"]
+	path = rust-agent/grpc-rs
+	url = https://github.com/alipay/grpc-rs.git
```

Comment:
@jodh-intel We do have some patch on `grpc-rs` and `grpc` itself (for example, vsock support). I think it may take a long time to land those patches if there are possible to land them all. Another option is to have fork repos of them in kata org, what do you think about this?

Reply:
Thanks @gnawux. Are there any open PRs on those two packages? It would be great if we could nudge those two projects to take the patches so we can drop the submodules :smile:

Reply:
@teawater your turn 🤣

Reply:
@jodh-intel 
https://github.com/grpc/grpc/pull/20272
This is the PR for add vsock to grpc.
It is blocked by a issue around CLA that @gnawux and me are working on it.

Reply:
The latest vsock PR: https://github.com/grpc/grpc/pull/21121.